core/src/apps/common/backup.py

```diff
@@ -0,0 +1,51 @@
+from typing import TYPE_CHECKING
+
+from trezor.enums import MessageType
+
+if TYPE_CHECKING:
+    from trezor.wire import Handler, Msg
+
+
+def repeated_backup_enabled() -> bool:
+    import storage.cache as storage_cache
+
+    return storage_cache.get_bool(storage_cache.APP_RECOVERY_REPEATED_BACKUP_UNLOCKED)
+
+
+def enable_repeated_backup():
+    import storage.cache as storage_cache
+
+    storage_cache.set_bool(storage_cache.APP_RECOVERY_REPEATED_BACKUP_UNLOCKED, True)
+
+
+def add_repeated_backup_filter():
+    from trezor import wire
+
+    wire.filters.append(_repeated_backup_filter)
+
+
+def disable_repeated_backup():
+    import storage.cache as storage_cache
+    from trezor import wire
```

Comment:
please move both those imports to top level.

both are (a) ok to move to top-level because we know that they're imported already (pretty much in all situations)
and (b) used often enough throughout this file that the top-level import is less expensive /in terms of bytecode size/ than repeating the same import statement multiple times

see https://github.com/trezor/upysize for some size considerations. but we don't really uphold that religiously, more like, from time to time we can run upysize on the codebase and see if there are savings to be had

Reply:
https://github.com/trezor/trezor-firmware/pull/3743/commits/95ed4817d8f4d90b6462e11c7d9aa77e278ab3cc